resolver/endpoint/rootca.go

```diff
@@ -0,0 +1,38 @@
+package endpoint
+
+import (
+	"crypto/x509"
+	"embed"
```

Comment:
In terms of binary size, wouldn’t embedding the cert in code make more sense?

Reply:
With `embed`, binary size if 9807719. By just dropping the certificate inside the source code, I have 9783885. I find cleaner to have certificates inside a directory in the source code, but we can drop them in code as well. As you prefer.

Reply:
Ok